src/yomu.ts

```diff
@@ -9,12 +9,24 @@ const cli = meow(
 
   Global Scope
     $ yomu -g <package-name>
+
+  Disable syntax highlight
+    $ yomu -c never <package-name>
+    $ yomu --color never <package-name>
```

Comment:
[Q.] Which do you think is better, --plain / -p?

Reply:
[IMO]
I think `--color / -c` better than `--plain / p`.
That's because option interface similar to ls(grep).
But I should probably change to not receive short option because, ls(grep) -c option is another meaning.

Reply:
話して別に `-p` でも良いことになった